source/_docs/configuration/templating.markdown

```diff
@@ -916,6 +916,23 @@ The numeric functions and filters raise an error if the input is not a valid num
 - Filter `value_one|bitwise_or(value_two)` perform a bitwise or(\|) operation with two values.
 - Filter `ord` will return for a string of length one an integer representing the Unicode code point of the character when the argument is a Unicode object, or the value of the byte when the argument is an 8-bit string.
 
+### Complex type checking
+
+In addition to strings and numbers, Python (and Jinja) support lists, sets, and dictionaries. To help you with testing these types, the following are available as functions, filters, and tests:
+
+- `is_list(x)` will return whether `x` is a `list` or not (e.g. [1, 2]).
```

Comment:
Please rewrite the example ("e.g. [1, 2]") is an example of a list, not of how `is_list` behaves.
Also, please explain how it works when used as a filter or test too.
Same comment for the other `is_xxx`` .

It's mentioned in the core PR, Jinja has a built in function called "mapping", can we mention it here too? Does that also allow use as a function, filter or test?